docs/docsite/rst/conf.py

```diff
@@ -240,3 +240,9 @@
 intersphinx_mapping = {'python': ('https://docs.python.org/2/', (None, '../python2-2.7.13.inv')),
                        'python3': ('https://docs.python.org/3/', (None, '../python3-3.6.2.inv')),
                        'jinja2': ('http://jinja.pocoo.org/docs/', (None, '../jinja2-2.9.7.inv'))}
+
+html_context = {
+    'current_version': '2.7',
```

Comment:
What's the impact if we have 'current_version': 'latest'  (so presumably we don't have to change that line ever?)